src/templates/layouts/healthCareLocalFacility/HealthServices/HealthServices.test.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { HealthServices } from './HealthServices'
+import { FormattedVAMCFacilityHealthService } from '@/types/formatted/healthCareLocalFacility'
+import { ParagraphServiceLocationAddress } from '@/types/drupal/paragraph'
+
+// Mock ServiceLocation to simplify tests
+jest.mock('./ServiceLocation', () => ({
+  ServiceLocation: jest.fn(() => <div>Mock ServiceLocation</div>),
+}))
+
+describe('HealthServices', () => {
+  const mockHealthServices: FormattedVAMCFacilityHealthService[] = [
+    {
+      name: 'Primary Care',
+      fieldAlsoKnownAs: 'General Medicine',
+      fieldCommonlyTreatedCondition: 'Diabetes, Hypertension',
+      description: '<p>Comprehensive primary care services</p>',
+      entityId: 1,
+      entityBundle: 'health_care_service',
+      locations: [
+        {
+          single: {
+            // We're mocking out the service location component, so we don't
+            // need to provide a full address object.
+            // NOTE: This is relying on static typing to test for the proper
+            // integration.
+            fieldServiceLocationAddress: {} as ParagraphServiceLocationAddress,
```

Comment:
I really didn't want to fill this out; the mock data here is so verbose as it is... 🫠